server/base/src/main/java/org/apache/accumulo/server/tabletserver/LargestFirstMemoryManager.java

```diff
@@ -292,7 +292,7 @@ protected long currentTimeMillis() {
   }
 
   @Override
-  public void tabletClosed(KeyExtent extent) {}
```

Comment:
Might other memory manager implementations need this?

Reply:
Yeah.  It is not easy to know what all is pluggable or just an internal interface.  I will revert